packages/mui-base/src/Portal/Portal.test.tsx

```diff
@@ -139,36 +139,36 @@ describe('<Portal />', () => {
 
       return (
         <span>
-          <strong ref={containerRef} />
-          <Portal disablePortal={disablePortal} container={container}>
+          <strong ref={containerRef as any} />
+          <Portal disablePortal={disablePortal} container={container as any}>
             <div id="test3" />
           </Portal>
         </span>
       );
     }
 
     const { setProps } = render(<ContainerTest />);
-    expect(document.querySelector('#test3').parentElement.nodeName).to.equal('SPAN');
+    expect((document.querySelector('#test3') as any).parentElement.nodeName).to.equal('SPAN');
```

Comment:
```suggestion
    expect(document.querySelector('#test3')?.parentElement.nodeName).to.equal('SPAN');
```

This should work, same for the other `expect` where you need to access the parent element

Reply:
I tried this already. It also resulted in the same issue: 
<img width="713" alt="Screen Shot 2022-11-03 at 01 19 39" src="https://user-images.githubusercontent.com/15035333/199674498-010dfd6d-2bc5-4a4a-bd5a-1585954d7238.png">

Reply:
```suggestion
    expect(document.querySelector('#test3')?.parentElement?.nodeName).to.equal('SPAN');
```

I forgot the 2nd one

Reply:
Except these, I removed every other 'any'. I will remember to use 'any' only as a last resort.

Reply:
just figured it out I added this change.

Reply:
Sorry my brain is shutting down its 1:50AM